libsolidity/ast/Types.cpp

```diff
@@ -1542,6 +1542,13 @@ MemberList::MemberMap ArrayType::nativeMembers(ContractDefinition const*) const
 				strings{string()},
 				isByteArray() ? FunctionType::Kind::ByteArrayPush : FunctionType::Kind::ArrayPush
 			)});
+			members.push_back({"pop", make_shared<FunctionType>(
```

Comment:
Beware: This is not inside the body of the if statement.

Reply:
Damn. Good catch! That's the reason why I prefer to have braces for ``if`` statements if the enclosed block spans over multiple (>2) lines. Enforced defect protection in a some way.